client/console/src/main/java/org/apache/syncope/client/console/init/ClassPathScanImplementationLookup.java

```diff
@@ -51,6 +53,8 @@
     private List<Class<? extends BaseExtPage>> extPages;
 
     private List<Class<? extends BaseExtWidget>> extWidgets;
+    
+    private List<String> basePackages = Arrays.asList("org.apache.syncope.client.console");
```

Comment:
I've implemented with a List<String> in order to have a similar implementation as in core.logic.init.ClassPathScanImplementationLookup.

Just tell me if you really want it to be a simple String, I'll change it.